modules/core/core-api/src/main/java/com/enonic/xp/project/ProjectPermissions.java

```diff
@@ -105,7 +103,7 @@ private Builder( final ProjectPermissions source )
 
         public Builder addOwner( final String owner )
         {
-            if ( StringUtils.isNotBlank( owner ) )
+            if ( owner != null )
```

Comment:
this is not exact conversion. Exact one is `!nullToEmpty( owner ).isBlank()`
But why do we need this null guard? What cand of caller tries to add `null` owner/editor/author/...?